modules/mariadb/src/main/java/org/testcontainers/containers/MariaDBContainer.java

```diff
@@ -17,8 +17,10 @@ public class MariaDBContainer<SELF extends MariaDBContainer<SELF>> extends JdbcD
 
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("mariadb");
 
+    private static final DockerImageName FOUNDATION_IMAGE_NAME = DockerImageName.parse("quay.io/repository/mariadb-foundation/mariadb-devel");
```

Comment:
`/repository/`  shouldn't be part of this. It won't resolve otherwise:

```
$ podman pull 'quay.io/repository/mariadb-foundation/mariadb-devel:11.7'
Trying to pull quay.io/repository/mariadb-foundation/mariadb-devel:11.7...
Error: initializing source docker://quay.io/repository/mariadb-foundation/mariadb-devel:11.7: reading manifest 11.7 in quay.io/repository/mariadb-foundation/mariadb-devel: unauthorized: access to the requested resource is not authorized
```